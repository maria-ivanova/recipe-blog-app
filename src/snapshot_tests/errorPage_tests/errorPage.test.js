import React from 'react';
import renderer from 'react-test-renderer';
import Routing from '../helpers/router.js';
import ErrorPage from '../../components/errorPage.js';

describe('Error page Component', () => {
    it('should render page layout', () => {
        const tree = renderer.create(
            <Routing>
                <ErrorPage />
            </Routing>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})