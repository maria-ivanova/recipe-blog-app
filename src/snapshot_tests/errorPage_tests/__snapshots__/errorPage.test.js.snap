// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error page Component should render page layout 1`] = `
<section
  className="sec content_sec"
>
  <div
    className="container tcenter"
  >
    <h2
      className="page_title"
    >
      <span>
        Грешка
      </span>
    </h2>
    <h3>
      Съжаляваме, но възникна грешка
    </h3>
  </div>
</section>
`;
